### PR TITLE
Hardware change

### PR DESCRIPTION
**Pull request by eSqadron**
overlay and driver initialisation adjusted to new, dual channel board

### CODE REVIEW

File: BrushedMotorControllerFirmware/boards/nrf52840dongle_nrf52840.overlay [hunk 1]
```diff
@@ -2,59 +2,67 @@
 	status="disabled";
 };
 
+//Hardware v 1.X
```
Comment (ghost) [inline BrushedMotorControllerFirmware/boards/nrf52840dongle_nrf52840.overlay:5]:
```suggestion
```

File: BrushedMotorControllerFirmware/include/driver.c [hunk 2]
```diff
@@ -236,7 +245,11 @@ static void enc_callback_wrapper(const struct device *dev, struct gpio_callback
 {
 	for (int channel = 0; channel < CONFIG_SUPPORTED_CHANNEL_NUMBER; ++channel) {
 		for (int pin = 0; pin < PINS_PER_CHANNEL; ++pin) {
-			if (pins & BIT(drv_chnls[channel].enc_pins[pin].pin)) {
+			if ((pins & BIT(drv_chnls[channel].enc_pins[pin].pin)) &&
+			    dev == drv_chnls[channel].enc_pins[pin].port) {
+
+				// printk("CH:%d, P:%d\n", channel, pin);
+
```
Comment (ghost) [inline BrushedMotorControllerFirmware/include/driver.c]:
```suggestion
```

File: BrushedMotorControllerFirmware/include/driver.c [hunk 3]
```diff
@@ -425,7 +438,7 @@ static int speed_pwm_set(uint32_t value, enum ChannelNumber chnl)
 		return ERR_DESIRED_VALUE_TO_HIGH;
 	}
 
-	if (drv_chnls[chnl].target_speed_mrpm < CONFIG_SPEED_MAX_MRPM / 10) {
+	if (drv_chnls[chnl].target_speed_mrpm < CONFIG_POS_CONTROL_MIN_SPEED_MODIFIER ) {
```
Comment (ghost) [inline BrushedMotorControllerFirmware/include/driver.c]:
```suggestion
	if (drv_chnls[chnl].target_speed_mrpm < CONFIG_POS_CONTROL_MIN_SPEED_MODIFIER) {
```

**Review by ghost — APPROVED** [review]
Few nits

File: BrushedMotorControllerFirmware/boards/nrf52840dongle_nrf52840.overlay (hunk 1 above)
Comment (eSqadron) [reply BrushedMotorControllerFirmware/boards/nrf52840dongle_nrf52840.overlay:5]: It's information printed on the board itself, I want to keep it in the overlay not to mix up overlays and hardwares

**Pull request merged**